src/Fable.Transforms/Python/Replacements.fs

```diff
@@ -2969,7 +2910,9 @@ let objects (com: ICompiler) (ctx: Context) r t (i: CallInfo) (thisArg: Expr opt
     match i.CompiledName, thisArg, args with
     | ".ctor", _, _ -> typedObjExpr t [] |> Some
     | "ToString", Some arg, _ -> toString com ctx r [ arg ] |> Some
-    | "ReferenceEquals", _, [ left; right ] -> makeEqOp r left right BinaryEqualStrict |> Some
+    | "ReferenceEquals", _, [ left; right ] ->
+        //makeEqOp r left right BinaryEqual |> Some
+        emitExpr None t [ left; right ] "$0 is $1" |> Some
```

Comment:
@alfonsogarciacaro Not sure what to do here. We cannot use `BinaryEqual` since that will compile to `==` and not `is`. And there is no way we can decide in Fable2Python if `BinaryEqual` should be `is` for any names. It must be `==` except for `ReferenceEquals`. Any ideas? I could make a `referenceEquals` function in `utils.py` to avoid the emit, but then the code is not as clean Python as it could have been. I'm unsure ...

Reply:
@dbrattli In theory you could do the same check to determine if it's a reference type in `Fable2Python` that was done before in `Replacements.Python`. The other option is to use some work-around by wrapping that `BinaryEqual` op in something (e.g. typecast to something like obj) so you can distinguish it from the regular equals.

Reply:
Yes, this is a pattern I use for some optimizations that happen after beta reduction. As @ncave says in Fable 3 I added a "tag" to TypeCast to mark this potential optimizations during the Replacements. However, later I felt this was stretching too much the purpose of TypeCast and in beyond I added an "OptimizableInto" optional tag to CallInfo. So you can emit a dummy call with OptimizableInto and catch it in Fable2Python. You can see how it's done in Fable2Babel/transformCall.

Reply:
@alfonsogarciacaro  Ok, I'll put it on the backlog, so please feel free to merge this when you feel for it.